operator/src/lib.rs

```diff
@@ -86,19 +87,26 @@ impl Operator {
         let wallet = EthereumWallet::from(ecdsa_signer.clone());
 
         //Create PubSubProvider
-        let ipc_path = "/tmp/anvil.ipc";
-        let ipc = IpcConnect::new(ipc_path.to_string());
-        let pubsub_provider: Arc<RootProvider<PubSubFrontend>> = Arc::new(
-            ProviderBuilder::new()
-                .on_ipc(ipc)
+        let pubsub_provider: Arc<RootProvider<PubSubFrontend>> = match chain {
+            Chain::Anvil => {
+                let ipc = IpcConnect::new("/tmp/anvil.ipc".to_string());
```

Comment:
Same as in the aggregator, we can use `ws_url` here

Reply:
I prefered to keep it separate, one is IPC connection the other is WS, the way the provider is built is different